Objective-C/Info.plist

```diff
@@ -19,8 +19,10 @@
 	<key>CFBundleVersion</key>
 	<string>$(CBL_BUILD_NUMBER)</string>
 	<key>NSHumanReadableCopyright</key>
-	<string>Copyright © 2017 Couchbase. All rights reserved.</string>
+	<string>$(CBL_COPYRIGHT)</string>
```

Comment:
I'm a bit reluctant to this change and it seems like it's safe to have the copy string in the Info.plist incase somebody grab the project and build by themselves without using the script.

Reply:
Can we do two things?
1. < string >Copyright © $(CBL_COPYRIGHT_YEAR) Couchbase. All rights reserved.< / string>
2. Add a build phase script to assign a default value to CBL_COPYRIGHT_YEAR variable if it doesn't have  the value? (NEED to try it out)